### DISCUSSION TITLE
Support daily notes without headings or any template at all

### DISCUSSION THREAD

**Discussion by yashk1996** [davish/obsidian-full-calendar#419] [Feature Requests]
Original bug report:
> When I try to add a daily note under some heading, instead of appending the heading in the daily note it says" Error while creating event, could not find heading in Daily note"

---

Updated issue description by @davish:
The `DailyNoteCalendar` relies on having a heading that it knows to put events under in daily notes. This heading is selected by a dropdown in settings by looking at what headings exist in the template note. If there is no template, there are no options for the heading to choose.

Daily note calendar settings should support the case where there are no headings in the template, or there is no template at all.

**Comment by davish** [comment]
Hi @yashk1996, thanks for submitting. I have a few questions that will help me reproduce this issue.
- What version of the plugin are you running?
- Are you using the core Daily Notes plugin, or Periodic Notes?
- Were you trying to add an event on a day where a daily note had already been generated?
- Could you post your daily note template here if you're comfortable doing that? After removing any personal info, of course.

**Comment by yashk1996** [comment]
Hi @davish thanks for the response. 
1. I'm using the latest version of the plugin: 0.10.5
2. I'm using Periodic Notes. However, I can confirm that it has the same problem with Daily Notes plugin.
3. Yes I was trying to add an event today(note existed already) for instance. However the issue still existed when I deleted the daily note. When I try to create the event in that case(daily note not created), it creates the Daily note of that date and shows the error message: can't find the heading in "date".md
4. Well actually I have no template. I don't use one at the moment.

**Comment by davish** [comment]
I think that point 4 might be the problem. I'll try to reproduce tonight, but I'll change the title of the issue in the meantime.

**Comment by yashk1996** [comment]
Yeah you're right. It works when I make a template with the heading.

**Comment by davish** [comment]
Changing this from a bug to an enhancement, since I don't believe this is supported in settings right now, either.
